e2e/_suites/ingest-manager/features/agent_endpoint_integration.feature

```diff
@@ -0,0 +1,31 @@
+@agent_endpoint_integration
+Feature: Fleet Mode Agent with Endpoint Integration
+  Scenarios for Agent to deploy Endpoint and sending data to Ingest Manager and Elasticsearch.
+
+@deploy-endpoint-with-agent
+Scenario: Deploy Endpoint with Agent and see host in Security App
+  Given an agent is deployed to Fleet
+  When the "Endpoint" latest package version is installed successfully
+    And a "Endpoint" package datasource is added to the 'default' configuration
+  Then the "default" configuration shows the "Endpoint" datasource added
+    And the host name is shown in the security app hosts list api
+    
+@endpoint-policy-check
+Scenario: Deploy Endpoint and see policy return successful in Security App
+  Given an Endpoint is successfully deployed with Agent
+  Then the policy response will be listed in the security app api
+
+@set-policy-and-check-changes
+Scenario: Deploy Endpoint and change default policy and verify in Security App
+  Given an Endpoint is successfully deployed with Agent
+  When the policy is updated to have malware in detect mode
+  Then the policy response with detect mode settings will be listed in the security app api
```

Comment:
If we have a reference to the policy response, could this step be rephrased to reuse the previous one?

```gherkin
  Then the policy response will be listed in the security app
```
Having "detect mode settings" is an attribute of the policy response, right?

Reply:
sounds good - thanks for the push to simplify and clean it up nicely